experimental/table/client/pom.xml

```diff
@@ -67,6 +67,11 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-all</artifactId>
```

Comment:
why netty is required?

Reply:
@HelloHorizon could you remove netty from both the client pom and the server pom, i don't think we need those.

Reply:
@yuzhu sure.